ambry-api/src/main/java/com/github/ambry/quota/Quota.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.quota;
+
+import com.github.ambry.quota.storage.QuotaOperation;
+import java.util.Collections;
+import java.util.Set;
+
+
+/**
+ * The quota for a particular Ambry resource.
+ * @param <T> the type of the quota value.
+ */
+public class Quota<T> {
```

Comment:
T should only be numeric types, like long, int, or float, right? If so, please put a comment here.

Reply:
done.